chart/templates/deployment.yaml

```diff
@@ -39,12 +42,19 @@ spec:
           httpGet:
             path: /ping
             port: {{ .Values.service.http.internalPort }}
-        volumeMounts:
-        - mountPath: /var/run/docker.sock
-          name: docker-socket
+      - name: dind-daemon 
+        image: docker:dind 
+        resources: 
```

Comment:
These were just carried over from an example. Anyone have input on what this should be set to?